app/views/components/_download-link.html.erb

```diff
@@ -0,0 +1,9 @@
+<%
+  download_link_text ||= "Download File"
+%>
+
+<div class="app-c-download-link">
```

Comment:
I think we were using the wrapper and/or making it block level so we could add spacing (margin-top) between the map and download map link. I'm not sure whether we want to keep this spacing or if this will be effected by our spacing changes later on?

Removing min-height does make the focus state smaller, but it still takes up a lot of the width of the page - is this expected behaviour?
<img width="658" alt="screen shot 2017-07-24 at 10 20 57" src="https://user-images.githubusercontent.com/29889908/28516702-078faa1a-705a-11e7-858e-edde47e561bd.png">